doc/whats_new/v0.22.rst

```diff
@@ -350,9 +350,13 @@ Changelog
   performance improvement. :pr:`13987` by `Roman Yurchak`_.
 
 - |Fix| KernelCenterer now throws error when fit on non-square
-  class:`preprocessing.KernelCenterer`
+  :class:`preprocessing.KernelCenterer`
   :pr:`14336` by :user:`Gregory Dexter <gdex1>`.
 
+- |Enhancement| label_binarize now supports list of lists for multilabel data
```

Comment:
Might be more helpful to advertise the changes in terms of how they affect metrics and other users of type_of_target

Reply:
how would you put it? one entry per impacted function or a more generic explanation on what is considered as multiclass-multioutput/multilabel-indicator?